azure/packages/test/end-to-end-tests/src/test/audience.spec.ts

```diff
@@ -19,13 +19,21 @@ const configProvider = (settings: Record<string, ConfigTypes>): IConfigProviderB
 	getRawConfig: (name: string): ConfigTypes => settings[name],
 });
 
+const isEphemeralContainer = process.env.IS_EPHEMERAL_CONTAINER === "true";
```

Comment:
Instead of adding this in every test file, I think you should put it in `AzureClientFactory.ts` in the same place all the other configs are read from the environment.

Reply:
Great idea! Also allows me to remove the undefined's from createAzureClient calls. Removed in recent commit